### DISCUSSION TITLE
Authentik and Squid Proxy Redirection

### DISCUSSION THREAD

**Discussion by Gamechiefx** [goauthentik/authentik#7678] [Q&A]
Hello I am looking to setup squid proxy to redirect to authentik login page for authentication. I am unsure what mechanism i can use within authentik to redirect the user to the squid proxy once the user has authenticated successfully. 

What would be the best approach to tackle this?

**Comment by isdnfan** [comment]
I don't think you can "redirect" proxy auth to an IdP. 

Proxy auth support AFAIK 3 protocols: basic auth, NTLM and Kerberos.. there is no way to add some SAML/OpenID provider.. but often proxy application can use LDAP as authentication backend using basic auth on the client side.

**Reply by bastiane273** [reply]
but you can block the access to the squid port - if you are not logged in into authentik
![image](https://github.com/goauthentik/authentik/assets/150721781/1d67160c-41b8-45d0-81a3-a9664a4e9706)
If you have traefik installed you can intercept the access to the squid port because of tcp port - entryPoint 
![image](https://github.com/goauthentik/authentik/assets/150721781/917eaab8-73ee-48d6-8176-09880d349f2c)

Here you are able to send automaticly an basic authentication - if you are logged in into authentic.
You have to use the Proxy Provider  - not OAUTH nor others
